src/Services/Calendar/TaskStore.vala

```diff
@@ -19,12 +19,12 @@
 */
 
 
-errordomain Tasks.TaskModelError {
+errordomain Calendar.TaskStoreError {
     CLIENT_NOT_AVAILABLE
 }
 
 
-public class Tasks.TaskModel : Object {
+public class Calendar.TaskStore : Object {
```

Comment:
Does it make sense to namespace this under Calendar for the Tasks app?

Reply:
Ideally the namespace should be `EDS` or similar for the Calendar as well as for the Tasks app. I did not change this yet, to avoid further confusion.